bench/perf_run_py/perf_stat_collection.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import os
+from perf_stat import PerfStat, PerfStatCount, PerfStatLatency, PerfStatMax, PerfStatMin, PerfStatLatencyWorkgen
+from typing import List
+
+
+def create_test_stat_path(test_home_path: str, test_stats_file: str):
+    return os.path.join(test_home_path, test_stats_file)
+
+
+class PerfStatCollection:
+    def __init__(self, operations: List[str]):
+        self.to_report: List[PerfStat] = []
+        if operations:
+            self.to_report = [stat for stat in PerfStatCollection.all_stats() if stat.short_label in operations]
+
+    def find_stats(self, test_home: str):
+        for stat in self.to_report:
+            test_stat_path = create_test_stat_path(test_home, stat.stat_file)
+            values = stat.find_stat(test_stat_path=test_stat_path)
+            stat.add_values(values=values)
+
+    @staticmethod
+    def all_stats():
+        return [
+            PerfStat(short_label="load",
```

Comment:
I was wondering if we could alphabetize this whole list, It would be easier to find stuff without calling grep!

Reply:
Yea, it's a good point, just that I am not very inclined towards changing the order of this list as it may break things in Atlas Charts.
Perhaps later, once the order is not relevant to Atlas? I am working on the very same.